README.md

```diff
@@ -6,14 +6,11 @@
 
 The Worldcoin Developer Portal provides tools to interact with the [World ID Protocol](https://worldcoin.org/world-id). Along with [IDKit](https://github.com/worldcoin/idkit-js), it's the fastest way to get started with proof of personhood 🚀
 
-<!-- WORLD-ID-SHARED-README-TAG:START - Do not remove or modify this section directly -->
```

Comment:
let's remove most of this and only keep link to the docs or it can become outdated quickly